crates/ty_project/src/metadata/options.rs

```diff
@@ -387,7 +391,7 @@ pub struct EnvironmentOptions {
             python = "./.venv"
         "#
     )]
-    pub python: Option<RelativePathBuf>,
+    pub python: Option<RangedValue<RelativePathBuf>>,
```

Comment:
This should not be necessary. `RelativePathBuf` wrapps `RangedValue` internally. But you might need to add a few methods to get the value (or decide that the current wrapping doesn't make any sense and rewrite all uses of `RelativePathBuf` to `RangedValue<RelativePathBuf>` and remove the `RangedValue` from `RelativePathBuf`)

Reply:
Thanks, I totally missed that